Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    File.exist?("~/Applications/Firefox.app")
+  end
```

Comment:
@jacknagel You're right. Symlinks are working with File.exist?. My fault. I've forgot about the path expanding.